python/tk_framework_desktopserver/shotgun/scripts/cache_commands.py

```diff
@@ -19,8 +21,8 @@ def cache(cache_file, data, base_configuration, hash_data, engine_name):
     project_entity = dict(id=data["project_id"], type="Project")
 
     # Setup the bootstrap manager.
-    toolkit_mgr = sgtk.bootstrap.ToolkitManager()
-    toolkit_mgr.plugin_id = "basic.shotgun.cache"
+    toolkit_mgr = sgtk.bootstrap.ToolkitManager(allow_config_overrides=False)
```

Comment:
this script should call [initialize_base_file_handler("tk-shotgun")](http://developer.shotgunsoftware.com/tk-core/utils.html#sgtk.log.LogManager.initialize_base_file_handler) as soon as it launches to make sure we capture as much of the log into to file as possible.